src/telephony/reducers.ts

```
@@ -1,12 +1,60 @@
+  };
+
+const normalizeTelephonyGlobalShortcutConfig = (
+  config: Partial<TelephonyGlobalShortcutConfig> | null | undefined
+): TelephonyGlobalShortcutConfig => {
+  if (!config || typeof config !== 'object') {
+    return defaultTelephonyGlobalShortcutConfig;
+  }
+
+  const accelerator = normalizeTelephonyShortcutAccelerator(
+    config.accelerator
+  );
```

ESLint

[error] Replace `⏎····config.accelerator⏎··` with `config.accelerator`

(prettier/prettier)